src/.babelrc

```diff
@@ -1,7 +1,7 @@
 {
   "compact": false,
   "presets": [
-    ["es2015", { "loose": true }],
+    ["env", { "loose": true }],
```

Comment:
We can't use `env` because source will be compiled according to the version of node installed on dev machine. Meanwhile, we should always compile for node 10.